app/helpers/distribution_helper.rb

```diff
@@ -17,4 +17,23 @@ def markdownify(text)
 
     markdown.render(text).html_safe
   end
+
+  def check_image_size(url)
+    conn = Faraday.new(url: url) do |f|
+      f.use FaradayMiddleware::FollowRedirects
+      f.request  :url_encoded
+      f.adapter  Faraday.default_adapter
+    end
+    # Turn into integer just in case we have dead links (they will report 0B)
+    conn.head.headers['content-length'].to_i
+  end
+
+  def check_short(short, image_size)
+    if short.nil?
```

Comment:
wrapping all the returns in the if block and then returning "short" at the end can be changed to a more readable way:
```ruby
  def short_description(short_desc, image_size)
    return short_desc if short_desc.present?

    case image_size
    when 0..700_000_000
      _("For CD and USB stick")
    when 700_000_001..5_000_000_000
      _("For DVD and USB stick")
    else
      _("For USB stick")
    end
  end
```